tests/stubtest_allowlists/py3_common.txt

```diff
@@ -165,6 +165,7 @@ poplib.POP3_SSL.stls  # bad declaration of inherited function. See poplib.pyi
 pyexpat.expat_CAPI
 select.poll  # Depends on configuration
 selectors.DevpollSelector  # Depends on configuration
+shutil.rmtree  # stubtest doesn't like that we have this as an instance of a callback protocol instead of a function
```

Comment:
I don't like that either. :/ But it's probably the best solution if it works.

Reply:
Yup, it's horrible. But I know of no other way to represent this in the type system :(